app/server/src/features/events/schema.graphql

```diff
@@ -187,6 +187,15 @@ type Query {
     events: [Event!]
 }
 
+"""
+Required to reduce frontend complexity due to relay limitation https://github.com/facebook/relay/issues/3457
+"""
+type EventEdgeContainer {
+    edge: EventEdge!
+}
+
 type Subscription {
-    eventUpdates(eventId: ID!): Event!
+    eventUpdates(userId: ID!): Event!
+    eventCreated(userId: ID!): EventEdgeContainer!
+    eventDeleted(eventIds: [ID!]!): EventEdgeContainer!
```

Comment:
You can remove these subscriptions and their resolvers since they are unused with the polling implementation.